src/lib/components/vegachart/VegaChart.component.js

```diff
@@ -5,37 +5,62 @@ import vegaEmbed from "vega-embed";
 
 type Props = {
   id: string,
-  spec: Object
+  spec: Object,
+  theme?: string // predefine theme light or dark
```

Comment:
`theme` could be light, dark (vega predefine theme) and anything else (custom...)
Please put the link 'How to theme tooltip here'